templates/import.ps1.erb

```diff
@@ -4,7 +4,7 @@ $cert = gi "<%= @location %>\<%= @name %>"
 
 switch -regex ($cert.Extension.ToUpper()) {
     ".CER|.DER|.P12" {
-        $pfx.Import("<%= @location %>\<%= @name %>","<%= @password %>","<%= @key_storage_flags %>")
+        $pfx.Import("<%= @location %>\<%= @name %>",'<%= @password %>',"<%= @key_storage_flags %>")
```

Comment:
What happens if the password contains a single quote?  I'm guessing that _may_ have worked before, but would be broken now.

Untested, but stdlib has a [powershell_escape](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/REFERENCE.md#powershell_escape) function, so something like...

```suggestion
       $pfx.Import("<%= @location %>\<%= @name %>",<%= scope.call_function('powershell_escape', [@password]) %>,"<%= @key_storage_flags %>")
```
should do the trick.  (It would mean bumping the minimum version of stdlib to `8.2.0` though)

Reply:
I tried upping the version of stdlib and using the powershell_escape function but got an unknown function error when testing. Finally got a regsubst function call through the linter and spec tests that will escape $ and " with back-ticks. Let me know if that works.